src/main/java/com/ubb/zenith/service/MoodService.java

```diff
@@ -0,0 +1,128 @@
+package com.ubb.zenith.service;
+
+import com.ubb.zenith.dto.MoodDTO;
+import com.ubb.zenith.exception.MoodAlreadyExistsExcetion;
+import com.ubb.zenith.exception.UserNotFoundException;
+import com.ubb.zenith.model.Mood;
+import com.ubb.zenith.repository.MoodRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class MoodService {
+
+    @Autowired
+    private MoodRepository moodRepository;
+
+    /**
+     *
+     * @return o lista de mood-uri
+     */
+    public List<Mood> getAll() {
+        return moodRepository.findAll();
+    }
+
+    /**
+     * verifica daca exista mood-ul daca se intampla acest lucru arunca o exceptie
+     * @param moodDTO
+     * @throws MoodAlreadyExistsExcetion
```

Comment:
After @throws MoodAlreadyExistsExcetion you can say when it throws MoodAlreadyExistsExcetion